docs/en/developers/set-autoscale.md

```diff
@@ -233,3 +124,143 @@ Introduce how to automatically scale workloads by cron in Appfile.
   ```
   
   Wait after the period ends, the replicas will be one eventually.
+
+## Setting auto-scaling policy of CPU resource utilization
+Introduce how to automatically scale workloads by CPU resource utilization.
+
+- Prepare Appfile
+  ```yaml
+  name: testapp
+
+  services:
+    express-server:
+      # this image will be used in both build and deploy steps
+      image: zzxwill/kubevela-appfile-demo:v1
```

Comment:
Can we reuse the previous appfile? So we just update it in this guide.

Reply:
Yes, just needed to add field `.autoscale.cron` and field `services.express-server.resources`. Will update this section.

And I was hesitated whether it's proper to specify the image `zzxwill/kubevela-appfile-demo:v1`. I intended to indicate developers should use a personal image here as it's told in https://github.com/oam-dev/kubevela/blob/master/docs/en/developers/devex/appfile.md#2-deploy-app-in-one-command. Or I should repeat the instructions as the appfile.md.

ATTENTION: change the image field in vela.yaml to something you can push to on your host:

> Or you may try the local testing option introduced in the following section.

```yaml
    image: oamdev/testapp:v1
```

Reply:
> And I was hesitated whether it's proper to specify the image zzxwill/kubevela-appfile-demo:v1

There's already an existing image `oamdev/testapp:v1`, why don't use it? Also, you don't need to define `build` if the image is already built ....

Reply:
Developers need write permission to push the image to docker hub per the requirement for building image in appfile.

Reply:
I got it. I should remove field `build` to use the `oamdev/testapp:v1` per the instruction from @wonderflow.

Reply:
I have actually removed build.
Please rebase and check :)

Reply:
> Add please also add reference doc for autoscaler like other traits.

Okay. I tried to automatically generate the docs but i haven't finished it. I will manually add it first. Thanks.